packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -8,41 +10,40 @@ import { TablePageObject } from './table.pageobject';
 interface SimpleTableRecord extends TableRecord {
     stringData: string;
     numericData: number;
-    booleanData: boolean;
-    dateData: Date;
+    booleanData: string;
+    moreStringData: string;
 }
 
-const simpleTableDataKeys = [
-    'stringData',
-    'numericData',
-    'booleanData',
-    'dateData'
-] as const;
-
 const simpleTableData = [
     {
         stringData: 'string 1',
         numericData: 8,
-        booleanData: true,
-        dateData: new Date(2008, 12, 11)
+        booleanData: 'true',
```

Comment:
Can we remove the `booleanData` field since it isn't actually a boolean?

Reply:
Removed.